misk/src/main/kotlin/misk/web/HttpCall.kt

```diff
@@ -11,13 +11,31 @@ import okhttp3.RequestBody
 import okio.BufferedSink
 import okio.BufferedSource
 
+/**
+ * Information about the socket on which a HTTP call arrived.
+ */
+data class SourceSocket (
+  val type: Type,
+  val name: String
```

Comment:
I think `name` is too weak. This would be worth properly modeling.

```
sealed class SocketAddress {
  class NetworkAddress(val ipAddress: String, val port: Int) : SocketAddress
  class UnixSocketAddress(val path: String) : SocketAddress
}
```

Reply:
(With sealed classes you also don’t need the enum!)